### DISCUSSION TITLE
langfuse-python: simple static method listing prompt template var names

### DISCUSSION THREAD

**Discussion by rhighs** [langfuse/langfuse#4414] [Ideas]
### Describe the feature or potential improvement

It would be nice to have a simple method call like `prompt.variables_names()` that returns a list of the var names present. For what I was able to see, we could reuse the [`_compile_template_string` static method.](https://github.com/langfuse/langfuse-python/blob/08298f6fb5b1c33905b5a786a4d1ab0de6cc2ce2/langfuse/model.py#L89).

I have some working code already and if you guys agree I could implement this change myself and submit a very short PR.

Bye 👋 

### Additional information

Example code

```python
@staticmethod
def variables_names(
    content: str
):
    opening = "{{"
    closing = "}}"
    curr_idx = 0
    result_list = []

    while curr_idx < len(content):
        var_start = content.find(opening, curr_idx)
        if var_start == -1:
            break
        var_end = content.find(closing, var_start)
        if var_end == -1:
            break
        variable_name = content[var_start + len(opening) : var_end].strip()
        result_list.append(variable_name)
        curr_idx = var_end + len(closing)

    return result_list

print(get_variables_names("""Explain how {{favorite_pizza_topping}} is linked to sightings of {{ufo_depth_in_meters}} beneath the ocean near {{research_base_name}}.
"""))
```

Expected output:

```
['favorite_pizza_topping', 'ufo_depth_in_meters', 'research_base_name']
```

**Comment by marcklingen** [comment]
@hassiebp what do you think? seems like a good extension of the prompt management sdk in my view

**Comment by hassiebp** [comment]
Great idea, @rhighs ! Feel free to submit a PR to support this 👍🏾

**Reply by rhighs** [reply]
Thank youuuuuuuu

**Comment by rhighs** [comment]
Reporting the PR here just for ref: https://github.com/langfuse/langfuse-python/pull/1013
